var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -64,6 +64,11 @@ class Libxsmm(MakefilePackage):
             description='With header-only installation')
     variant('generator', default=False,
             description='With generator executable(s)')
+    variant('noblas', default=False,
+            description='LIBXSMM without GEMM-related functionality')
```

Comment:
```suggestion
    variant('blas', default=True,
            description='LIBXSMM with GEMM-related functionality')
```
Let's avoid double negatives, it gets more confusing that way

Reply:
I agree, @hfp, can you comment on the best approach at exposing the _BLAS=_ Makefile option as a variant here?
It is unclear to me what the default value is (i see mentions of 1, and 2). Additionally, if it is set to 1 or 2, does it require a BLAS dependency ( i do not see one currently). Thanks

Reply:
I have made changes in line with what @adamjstewart  mentioned. @hfp can you give the OK on this?